src/parsley/parse_to_object.py

```
@@ -0,0 +1,264 @@
+            raise ValueError(f'Bad checksum, expected {exp_crc:02X} but got {msg_crc:02X}')
+
+        return _ParsleyParseInternal.parse_to_object(msg_sid, list(msg_data))
+
+class LoggerParser(ParsleyParser):
+    """ Parses logger pages and yields `ParsleyObject` items """
+    
+    """
+    Parse one logger record.
+
+    Layout  (little-endian unless stated):
+        0  – 2  : ASCII 'L','O','G'
```

Ruff (0.14.10)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)